### PR TITLE
Compile: fix make package failure

### PR DESCRIPTION
**Pull request by liangzhou121**
Signed-off-by: Zhou Liang <liang.a.zhou@linux.alibaba.com>

### CODE REVIEW

File: dist/deb/debian/control [hunk 1]
```diff
@@ -4,7 +4,7 @@ Priority: extra
 Maintainer: Zhou Liang <liang.a.zhou@linux.alibaba.com>
 Build-Depends: debhelper (>=9), cmake, git, make, autoconf, libtool, gcc, g++, libsgx-dcap-quote-verify-dev, libsgx-dcap-ql-dev, libsgx-uae-service
 Standards-Version: 3.9.8
-Homepage: https://github.com/alibaba/inclavare-containers
+Homepage: https://github.com/alibaba/inclavare-containers/verdictd
```
Comment (haosanzi) [inline dist/deb/debian/control:7]: Please change to `https://github.com/inclavare-containers/verdictd`

File: dist/rpm/verdictd.spec.in [hunk 2]
```diff
@@ -14,8 +14,8 @@ Summary: verdictd is the Key Broker Service (KBS) of Enclave Attestation Archite
 
 Group: Development/Tools
 License: Apache License 2.0
-URL: https://github.com/alibaba/%{PROJECT}
-Source0: https://github.com/alibaba/%{PROJECT}/archive/v%{version}.tar.gz
+URL: https://github.com/alibaba/inclavare-containers
+Source0: https://github.com/alibaba/inclavare-containers/archive/v%{version}.tar.gz
```
Comment (haosanzi) [inline dist/rpm/verdictd.spec.in:18]: ditto

**Comment by liangzhou121** [comment]
Please don't merge this PR

File: dist/Makefile [hunk 3]
```diff
@@ -6,7 +6,7 @@ ifeq ($(VERDICTD_MAINTAINER),)
 $(error "Please run make package in the root directory")
 endif
 
-PROJECT_DIR := $(shell cd ../..; pwd)
+PROJECT_DIR := $(shell cd ../; pwd)
```
Comment (jiazhang0) [inline dist/Makefile:9]: Use this to set the root directory of priject

```
PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
```

**Pull request closed without merging**